hooks/mari/collector.py

```diff
@@ -42,6 +42,10 @@
     "mari.texture": {
         "icon_path": "{self}/hooks/icons/texture.png",
         "type_display": "Layer"    
+    },
+    "mari.mipmap": {
+        "icon_path": "{self}/hooks/icons/mari_mipmap.png",
```

Comment:
Does this path exist?

Reply:
I copied over a grid icon and added it (it's on git in this branch now). It seemed appropriate.

Reply:
New (and improved?) icon added :P